packages/decoder/package.json

```diff
@@ -41,7 +41,7 @@
     "@types/debug": "^0.0.31",
     "@types/web3": "1.0.19",
     "chai": "^4.2.0",
-    "ganache-core": "^2.13.1",
+    "ganache-core": "2.13.0",
```

Comment:
Why'd you pin this?

Reply:
I mentioned it at the end -- just consistency, everything else was pinned to 2.13.0.  But instead I could go and pin everything to 2.13.1 so they're all upgraded. :)

Reply:
Ah sorry, missed that. Probably upgrade and unpin, but whatever, this makes sense for this PR.